WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingHomeViewModel.kt

```diff
@@ -43,16 +49,18 @@ class PrepublishingHomeViewModel @Inject constructor(
                         },
                         onActionClicked = ::onActionClicked
                 ),
-                PrepublishingHomeUiState(
+                HomeUiState(
                         actionType = VISIBILITY,
                         actionResult = UiStringText("Public"),
                         onActionClicked = ::onActionClicked
                 ),
-                PrepublishingHomeUiState(
+                HomeUiState(
                         actionType = TAGS,
-                        actionResult = getPostTagsUseCase.getTags(editPostRepository)?.let { UiStringText(it) },
+                        actionResult = getPostTagsUseCase.getTags(editPostRepository)?.let { UiStringText(it) }
+                                ?: run { UiStringRes(R.string.prepublishing_nudges_home_tags_not_set) },
                         onActionClicked = ::onActionClicked
-                )
+                ),
+                PublishButtonUiState(UiStringRes(R.string.prepublishing_nudges_home_publish_button), {})
```

Comment:
I assume we are passing an empty onClickListener since this logic isn't implemented yet, right?

Reply:
Yes, this is being done here https://github.com/wordpress-mobile/WordPress-Android/pull/11805